artistic-style-check.sh

```diff
@@ -48,7 +48,9 @@ while IFS='' read -r targetPath && [[ -n "$targetPath" ]]; do
     # Check if it's a file (find matches on pruned folders)
     if [[ -f "$filename" ]]; then
       echo "::group::Checking $filename"
-      if ! diff --strip-trailing-cr "$filename" <(astyle --options="$optionsFilePath" --dry-run <"$filename"); then
+      diffOutput=$(diff --strip-trailing-cr "$filename" <(astyle --options="$optionsFilePath" --dry-run <"$filename"))
+      if [[ -n "$diffOutput" ]]; then
+        echo "$diffOutput"
```

Comment:
@danwahl please limit the scope of the changes in the PR exclusively to the stated purpose of fixing the download URL. You can submit separate PRs for other proposals.